src/serviceregistry/manual/examples.md

```diff
@@ -178,13 +184,9 @@ This file contains specific examples of what payloads are expected and returned
     },
     "version": "2.1.0",
     "addresses": [
-      {
-        "type": "MAC",
-        "address": "a1:b2:c3:34:55:f6"
-      },
       {
         "type": "IPV4",
-        "address": "127.77.66.1"
+        "address": "192.168.1.2"
```

Comment:
This Ip is not the same than in was specified in the request. Also, missing the hostname.